form.js

```diff
@@ -0,0 +1,25 @@
+function validation() {
```

Comment:
The error should be descriptive when the user fills the email with the upper case. The current error message is `*Make sure to fill your name` even all the fields are filled. Kindly see the screenshot below and fix the issue.
![upper](https://user-images.githubusercontent.com/35504149/132946757-79fcec9b-89f8-4dfc-80ce-d3de03d49782.png)